tests/models/test_farseg.py

```diff
@@ -7,36 +7,26 @@
 
 from torchgeo.models import FarSeg
 
-BACKBONE = ["resnet18", "resnet34", "resnet50", "resnet101", "anynet"]
-
 
 class TestFarSeg:
     @torch.no_grad()  # type: ignore[misc]
-    def test_classes(self) -> None:
```

Comment:
Old tests had a lot of redundancy, new tests get more coverage by testing both pretrained True and False